### PR TITLE
Collaborative gat update branch

### PR DESCRIPTION
**Pull request by hexylena**
Potential things to address

- [x] ~https://github.com/galaxyproject/galaxy/pull/15131~ #3975 
- [x] https://github.com/galaxyproject/ansible-galaxy/pull/169
- [x] https://github.com/galaxyproject/training-material/issues/2583
- [x] https://github.com/galaxyproject/training-material/issues/1809
- [x] https://github.com/galaxyproject/training-material/pull/3654

Fixes https://github.com/galaxyproject/training-material/issues/3665

### CODE REVIEW

**Comment by hexylena** [comment]
@natefoo @mira-miracoli @cat-bro I've updated the deployment schema here. If you have any other misc changes y'all want to get in feel free to commit directly to this branch.

File: topics/admin/tutorials/backup-cleanup/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+- Learn what to back up and how to recover
+time_estimation: "30m"
+key_points:
+  - Use configuration management (e.g. Ansible)
+  - Store configuration management in git
+  - Back up the parts of Galaxy that can't be recreated
+contributions:
+  authorship:
+  - hexylena
+  - natefoo
```
Comment (hexylena) [inline topics/admin/tutorials/backup-cleanup/tutorial.md:24]: @natefoo can you look over this one briefly?

We dont' discuss restoration, but I know you had documentation for that in a PR, could you link that here and I'll write a small section on restoration?

Comment (lldelisle) [reply topics/admin/tutorials/backup-cleanup/tutorial.md:24]: Here is the link: https://github.com/galaxyproject/ansible-postgresql/pull/30#issuecomment-963600656
And here is how I struggled but managed to do it:
https://github.com/lldelisle/galaxyduboule-infrastructure/blob/778e5b056f03970a1af5fc2f9dd133b4e4791cac/testUpgradeOnVM.sh#L53-L130

Comment (hexylena) [reply topics/admin/tutorials/backup-cleanup/tutorial.md:24]: oh!!! amazing. I owe you a drink next time we meet :) this saves so much time

Comment (lldelisle) [reply topics/admin/tutorials/backup-cleanup/tutorial.md:24]: You see I am a good beta-tester... :wink:

**Comment by hexylena** [comment]
Ah, perfect! please feel free to commit directly to this branch! :)

**Comment by mira-miracoli** [comment]
hm since I did not find an ansible role to install apptainer, only SingularityCE, maybe we should stick to the name and keep synlabs fork for now

**Comment by hexylena** [comment]
> hm since I did not find an ansible role to install apptainer, only SingularityCE

Is the install process so different though? We could surely use the existing role with the new github url right?

> maybe we should stick to the name and keep synlabs fork for now

Fine for me. I think at minimum we should have a small table explaining the three different forks and their differences. Are you familiar enough with them to write that?

**Comment by mira-miracoli** [comment]
> Is the install process so different though? We could surely use the existing role with the new github url right?

Yes it is a different installation process. https://github.com/CyVerse-Ansible/ansible-singularity/blob/master/tasks/singularity-post-3.yml
For RHEL systems, you can simply install it with `dnf install apptainer`
But I think it is different for Ubuntu
according to my google research:
```
sudo add-apt-repository -y ppa:apptainer/ppa
sudo apt install -y apptainer
```

**Comment by mira-miracoli** [comment]
Okay, shall I make a role for that? Do we switch to Apptainer than?

**Comment by hexylena** [comment]
I'm already on it, but yes, let's switch.

**Pull request merged**